src/libs/PusherConnectionManager.js

```diff
@@ -16,16 +17,20 @@ function init() {
     }));
 
     /**
-     * Events that happen on the pusher socket are used to determine if the app is online or offline.
-     * The offline setting is stored in Onyx so the rest of the app has access to it.
-     *
      * @params {string} eventName
      */
-    Pusher.registerSocketEventCallback((eventName, data) => {
+    Pusher.registerSocketEventCallback((eventName, error) => {
         switch (eventName) {
             case 'error':
-                Log.info('[PusherConnectionManager] error event', false, {error: data});
-                Session.reauthenticatePusher();
+                if (error && error.type === 'PusherError' && error.data.code === 1006) {
+                    Log.hmmm('[PusherConnectionManager] Channels Error 1006', {error});
+                } else if (error && error.type === 'PusherError' && error.data.code === 4201) {
```

Comment:
Is there a place where these seemingly-arbituary integer error codes are referenced?

Reply:
(aka could we add this as a comment to reference?)

Reply:
Yeah, I think I can do that.